packages/keyring-snap-bridge/src/SnapKeyring.ts

```diff
@@ -126,6 +129,8 @@ export type SnapKeyringCallbacks = {
     handleUserInput: (accepted: boolean) => Promise<void>,
   ): Promise<void>;
 
+  getSelectedAccounts(snapId: SnapId): Promise<GetSelectedAccountsResponse>;
+
```

Comment:
I think we can remove this if you're ok with my idea of keeping an in-memory list of selected account IDs within the `SnapKeyring` instance (the main reason I want to avoid adding a callback is because this would be considered a breaking change, and thus, would normally require aligning deps on the `core` side too, if we want to be "strict" with our dependencies management ofc 😅)